spark/src/test/scala/ai/chronon/spark/test/LabelJoinTest.scala

```diff
@@ -23,12 +23,14 @@ import org.apache.spark.sql.{Row, SparkSession}
 import org.junit.Assert.assertEquals
 import org.junit.Test
 
+import scala.util.Random
+
 class LabelJoinTest {
   @transient private lazy val logger = LoggerFactory.getLogger(getClass)
 
   val spark: SparkSession = SparkSessionBuilder.build("LabelJoinTest", local = true)
 
-  private val namespace = "label_join"
+  private val namespace = "label_join" + "_" + Random.alphanumeric.take(6).mkString
```

Comment:
might need to update hard coded references

https://sourcegraph.d.musta.ch/github.com/airbnb/chronon/-/blob/spark/src/test/scala/ai/chronon/spark/test/LabelJoinTest.scala?L59

https://sourcegraph.d.musta.ch/github.com/airbnb/chronon/-/blob/spark/src/test/scala/ai/chronon/spark/test/LabelJoinTest.scala?L104-111

https://sourcegraph.d.musta.ch/github.com/airbnb/chronon/-/blob/spark/src/test/scala/ai/chronon/spark/test/LabelJoinTest.scala?L355-356